makefiles/docker.inc.mk

```
@@ -39,9 +39,9 @@ ifeq (0,$(INSIDE_DOCKER))
       $(warning Required docker image $(DOCKER_IMAGE) not installed)
       ifeq (1,$(DOCKER_AUTO_PULL))
         $(info Pulling required image now automatically. You can disable this with DOCKER_AUTO_PULL=0)
-        $(shell $(DOCKER) pull $(DOCKER_REPO)@sha256:$(DOCKER_TESTED_IMAGE_REPO_DIGEST))
+        $(shell $(DOCKER) pull '$(DOCKER_REPO)@sha256:$(DOCKER_TESTED_IMAGE_REPO_DIGEST)')
       else
-        $(info Building with latest available riotbuild image. You can pull the correct image automaticale with DOCKER_AUTO_PULL=0)
+        $(info Building with latest available riotbuild image. You can pull the correct image automatically with DOCKER_AUTO_PULL=0)
```

```suggestion
        $(info Building with latest available riotbuild image. You can pull the correct image automatically with DOCKER_AUTO_PULL=1)
```

Sorry, you were too fast :P